CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Removed
 
 ### Fixed
+- Calculation of selector precedence for selectors involving pseudo-classes
+  and/or attributes.
```

Comment:
Could you please add a link to the PR (so this is consistent with the other changelog entries)? Then we don't need to edit your PR before we merge it. Thanks!

Reply:
Done